kvdb/postgres/readwrite_tx.go

```diff
@@ -48,10 +52,39 @@ func newReadWriteTx(db *db, readOnly bool) (*readWriteTx, error) {
 	}, nil
 }
 
-// rooBucket is a helper function to return the always present
-// pseudo root bucket.
-func rootBucket(tx *readWriteTx) *readWriteBucket {
-	return newReadWriteBucket(tx, "kv", nil)
+func hasSpecialChars(s string) bool {
+	for _, b := range s {
+		if !(b >= 'a' && b <= 'z') &&
+			!(b >= '0' && b <= '9') && b != '-' {
+
+			return true
+		}
+	}
+
+	return false
+}
+
+func toTableName(key []byte) string {
+	// TODO: Find different solution to deal with potential collisions.
+	hashedKey := sha256.Sum256(key)
+	key = hashedKey[:]
```

Comment:
Won't this make it so that every table now has a hex name?

Reply:
Somehow it seems you reviewed outdated code, even though I pushed the latest already yesterday. Sha256 is no longer happening.